src/host.cc

```diff
@@ -2311,7 +2311,7 @@ void host::check_for_expired_acknowledgement() {
 //  }
 //
 //  /* check notification dependencies */
-//  if (check_dependencies(hostdependency::notification) ==
+//  if (authorized_by_dependencies(hostdependency::notification) ==
 //      DEPENDENCIES_FAILED) {
 //    logger(dbg_notifications, more)
 //        << "Notification dependencies for this host have failed, "
```

Comment:
you can delete this commented code.

Reply:
Yes but let's wait a little more before. It is in that code that I found what I had erased...

Reply:
It's a treasure!